### PR TITLE
Ynab csv converter 20 automate csv download from poalim

### PR DESCRIPTION
**Pull request by itay101**
In this PR:

1.  npm was added for using playwright as an automation library.
2. Password and username should be added to each account in the config.json file
3. main.js is the main script for running the automation and then main.py is triggered for processing everything
4. Since I created this script as bash alias, I changed the paths to be absolute from Users/<user>/Downloads

### CODE REVIEW

File: files_processors/poalim_processor.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def __init__(self, **kwargs):
 
     @staticmethod
     def identify_account(file, accounts=[]):
-        if file.name.startswith("shekel"):
+        if "poalim" in file.name:
```
Comment (ventorero) [inline files_processors/poalim_processor.py]: maybe adding it under a variable ?

File: main.py [hunk 2]
```diff
@@ -4,28 +4,27 @@
 from common.utils import ynab_api
 from common.utils import config_api
 from enums import AccountTypeToProcessor
-from enums import AccountTypes
-from files_processors.raw_files import RawCSVFile
 
-CONFIG_FILE_PATH = "config.json"
+CONFIG_FILE_PATH = "~/code/ynab-csv-converter/config.json"
```
Comment (ventorero) [inline main.py:8]: relative path does not work here ?

File: main.py [hunk 3]
```diff
@@ -4,28 +4,27 @@
...
-
 def process_files():
-    config_file = open(CONFIG_FILE_PATH)
+    config_file = open(os.path.expanduser(CONFIG_FILE_PATH))
     config_data = json.load(config_file)
     accounts = config_data["accounts"]
     transactions = []
     files_added = []
-    for _root, _dirs, files in os.walk("./"):
+    for _root, _dirs, files in os.walk(os.path.expanduser("~/Downloads")):
```
Comment (ventorero) [inline main.py:19]: maybe /tmp ?

File: tests/poalim.spec.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { test, expect } from '@playwright/test';
+
+test('test', async ({ page }) => {
```
Comment (ventorero) [inline tests/poalim.spec.ts]: wowwwww what a test 👑

File: main.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const {exec} = require("child_process");
+
+const rawdata = JSON.parse(fs.readFileSync('config.json'));
+const ACCOUNT_TYPE_TO_SPEC_FILE = {
+    poalim: "poalim.spec.ts"
```
Comment (ventorero) [inline main.js]: is it possible to move it to enums  ?

**Review by ventorero — APPROVED** [review]
approved with comments

File: main.js (hunk 5 above)
Comment (itay101) [reply main.js]: I think of removing this and rely on the type of the account for the spec file

File: main.py (hunk 3 above)
Comment (itay101) [reply main.py:19]: Files from the browser are downloaded to the "Downloads" folder eventually

File: main.py (hunk 2 above)
Comment (itay101) [reply main.py:8]: The relative path doesn't do well anymore when you set the main.js script as a shell script alias. You can run it from everywhere

**Pull request merged**